zuul.d/edpm.yaml

```diff
@@ -19,6 +19,12 @@
     abstract: true
     irrelevant-files:
```

Comment:
wondering if we should get "files" and run only if ci_framework/* is changed? Seems a bit more suitable.. WDYT?

Reply:
This job template is used to run edpm job in idividual repo, and we have to run it in any logic change.